src/components/BookForm.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/books/books';
+
+const BookForm = () => {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
+  const [category, setCategory] = useState('');
+
+  const addNewBook = () => {
+    const newBook = {
+      title,
+      category,
+      id: uuidv4(),
+    };
+
+    dispatch(addBook(newBook));
+    setTitle('');
+    setCategory('');
+  };
+  const handelTitleChange = (e) => {
+    setTitle(e.target.value);
+  };
+  const handelCategoryChange = (e) => {
+    setCategory(e.target.value);
+  };
+  return (
+    <div>
+      <h5>ADD NEW BOOK</h5>
+      <input
+        placeholder="Book Title"
+        type="text"
+        onChange={handelTitleChange}
+        value={title}
+        required
+      />
+      <select
+        id="cars"
+        name="cars"
```

Comment:
[OPTIONAL] I think `cars` might not be the more descriptive id and name for this input. Sometimes writing a bit more can save some headaches in the future when we need to revisit our code. It would be nice if we come up with a more descriptive id here, like: 'categories'. But feel free to use another one you like.